pkg/apis/configuration/validation/transportserver_test.go

```diff
@@ -206,6 +206,122 @@ func TestValidateTransportServerHost(t *testing.T) {
 	}
 }
 
+func TestValidateTransportServerLoadBalancingMethod(t *testing.T) {
+	tests := []struct {
+		method   string
+		isPlus   bool
+		hasError bool
+	}{
+		{
+			method:   "",
+			isPlus:   false,
+			hasError: false,
+		},
+		{
+			method:   "",
+			isPlus:   true,
+			hasError: false,
+		},
+		{
+			method:   "hash",
+			isPlus:   false,
+			hasError: true,
+		},
+		{
+			method:   "hash ${remote_addr}",
+			isPlus:   false,
+			hasError: false,
+		},
+		{
+			method:   "hash ${remote_addr}AAA",
```

Comment:
Could you add one more case
```
		},
		{
			method:   `hash ${remote_addr}"`,
			isPlus:   false,
			hasError: true,
		},
```
This will cover `if !escapedStringsFmtRegexp.MatchString(method) {`

Reply:
done